liquidprompt

```diff
@@ -3376,6 +3527,13 @@ _lp_default_theme_prompt_data() {
         LP_VCS=
     fi
 
+    # Should be called after the dev env group above.
+    if _lp_dev_env_color ; then
```

Comment:
Now that the sections are no longer used by default in the theme, Shellcheck is yelling about them. Add `# shellcheck disable=SC2034` the line above `_lp_default_theme_prompt_data() {` to fix that.